### PR TITLE
Add if-else statement to run tests against real website on schedule

### PR DESCRIPTION
**Pull request by XiangRongLin**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I have tested the API against [NewPipe](https://github.com/TeamNewPipe/NewPipe).
- [x] I agree to create a pull request for [NewPipe](https://github.com/TeamNewPipe/NewPipe) as soon as possible to make it compatible with the changed API.

I realized that running the tests against real website is basically just copying the ci.yml and changing one value.
Which can also be done with an if condition.

Because github actions `if` has no `else`, the shell script ones are used

### CODE REVIEW

**Comment by Stypox** [comment]
I like this! Is it really that simple? :-D

**Comment by XiangRongLin** [comment]
Should be.
I forgot to link this one: https://github.com/XiangRongLin/NewPipeExtractor/pull/4
where the action does: https://github.com/XiangRongLin/NewPipeExtractor/pull/4/checks

**Comment by XiangRongLin** [comment]
@Stypox let the action on my repo run for a night and tomorrow we can see if it echoed the correct thing

**Comment by XiangRongLin** [comment]
Seeing how there already are new failing tests not even an hour after merging #512, we probably need a way to mark tests to be skipped in the ci pipeline if they are not run with mocks.

**Comment by Stypox** [comment]
@XiangRongLin good idea

**Comment by XiangRongLin** [comment]
> let the action on my repo run for a night and tomorrow we can see if it echoed the correct thing

It works
https://github.com/XiangRongLin/NewPipeExtractor/runs/1711997487?check_suite_focus=true#step:2:8

**Comment by Stypox** [comment]
Great! @TobiGr

File: .github/workflows/ci.yml
Comment (TobiGr) [inline .github/workflows/ci.yml:5]: Once a day? Coud you add that as a comment, so it is easeir to understand?

File: .github/workflows/ci.yml
Comment (TobiGr) [inline .github/workflows/ci.yml:30]:
```suggestion
        # See gradle file for difference between downloaders
```

**Review by TobiGr — COMMENTED** [review]
Can be merged once review is implemented

File: .github/workflows/ci.yml
Comment (XiangRongLin) [reply .github/workflows/ci.yml:6]: done

File: .github/workflows/ci.yml
Comment (XiangRongLin) [reply .github/workflows/ci.yml]: done

**Review by TobiGr — APPROVED** [review]

**Comment by TobiGr** [comment]
Ah wait. wasn't there a complaint regarding automated testing when they took down youtube-dl?

@TheAssassin This introduces sheduled tests against the real websites, whereas some predefined JSONs are tested against PRs. Is that ok?

**Comment by XiangRongLin** [comment]
@TobiGr I think part of it was that there we unit tests, that were downloading copyrighted stuff

see last paragraph of this section https://github.blog/2020-11-16-standing-up-for-developers-youtube-dl-is-back/#youtube-dl

So we would need to check all tests once and see if there are any such tests

**Pull request merged**
